nordicsemi/__main__.py

```diff
@@ -977,7 +977,7 @@ def thread(package, port, address, server_port, panid, channel, jlink_snr, flash
         click.echo("Waiting for NCP to promote to a router...")
         time.sleep(6.0)
         dfu.trigger(address, 3)
-        click.echo("Thread DFU server is running... Press <Ctrl + C> to stop.")
+        click.echo("Thread DFU server is running... Press <Ctrl + D> to stop.")
```

Comment:
Was this change intended? Is there any situation where `Ctrl+C` doesn't send a `SIGTERM` to the nrfutil process?

Reply:
Ctrl+C sends SIGINT and it's trapped somewhere before our signal handling because it doesn't terminate the process. The Ctrl+D does work.